src/rust/src/declarative_asn1/types.rs

```diff
@@ -66,13 +67,42 @@ pub struct AnnotatedTypeObject<'a> {
 
 #[pyo3::pyclass(module = "cryptography.hazmat.bindings._rust.asn1")]
 #[derive(Clone, Debug)]
-pub struct Annotation {}
+pub struct Annotation {
+    #[pyo3(get)]
+    pub(crate) default: Option<Default>,
+}
 
 #[pyo3::pymethods]
 impl Annotation {
     #[new]
-    fn new() -> Self {
-        Self {}
+    #[pyo3(signature = (default = None))]
+    fn new(default: Option<Default>) -> Self {
+        Self { default }
+    }
+}
+
+#[pyo3::pyclass(module = "cryptography.hazmat.bindings._rust.asn1")]
+#[derive(Debug)]
+pub struct Default {
+    #[pyo3(set, get)]
+    pub value: pyo3::Py<pyo3::types::PyAny>,
+}
+
+#[pyo3::pymethods]
+impl Default {
+    #[new]
+    #[pyo3(signature = (value))]
+    fn new(value: pyo3::Py<pyo3::types::PyAny>) -> Self {
+        Self { value }
+    }
+}
+
+impl Clone for Default {
```

Comment:
Why do we need this?

Reply:
For a gettable attribute, we should just store it as `Py<Default>`

Reply:
fixed